lms/templates/courseware/progress.html

```diff
@@ -168,21 +169,26 @@ <h3 class="hd hd-3" id="chapter_${loop.index}">${ chapter['display_name']}</h3>
                                 %for section in chapter['sections']:
                                     <div>
                                         <%
-                                        earned = section.all_total.earned
-                                        total = section.all_total.possible
-
-                                        percentageString = "{0:.0%}".format(section.percent_graded) if earned > 0 and total > 0 else ""
+                                        if section.override:
+                                            earned = section.override.earned_graded_override
+                                            total = section.override.possible_graded_override
+                                        else:
+                                            earned = section.all_total.earned
```

Comment:
I know the reference to `all_total` existed before you edited this code...I wonder if it makes more sense to refer to `graded_total` here instead?